crates/compiler/can/src/expr.rs

```diff
@@ -1652,7 +1651,7 @@ fn canonicalize_var_lookup(
 }
 
 /// Currently uses the heuristic of "only inline if it's a builtin"
-pub fn inline_calls(var_store: &mut VarStore, scope: &mut Scope, expr: Expr) -> Expr {
+pub fn inline_calls(var_store: &mut VarStore, expr: Expr) -> Expr {
```

Comment:
What a great catch, super nice that they made this a warning by default now

Reply:
yes it's amazing

Reply:
Wow, "argument is only used in recursive calls" I guess? That's awesome - we should do that! 😍